MOM6_config/OM4_025/.ipynb_checkpoints/MOM_override-checkpoint

```diff
@@ -0,0 +1,9 @@
+! Blank file in which we can put "overrides" for parameters
+!VERBOSITY = 9
+EKE_SOURCE='silc'
+SILC_MODEL='ncar_ml_eke.gpu.pt'
```

Comment:
change all these refereces to ``SILC``

Reply:
This should be deleted...it's a super old notebook thing.